arcgis-ios-sdk-samples/Scenes/Realistic lighting and shadows/RealisticLightingAndShadows.storyboard

```diff
@@ -0,0 +1,104 @@
+                                                    <rect key="frame" x="0.0" y="0.0" width="240" height="36"/>
+                                                    <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                                    <nil key="textColor"/>
+                                                    <nil key="highlightedColor"/>
+                                                </label>
+                                            </subviews>
+                                            <constraints>
+                                                <constraint firstAttribute="trailing" secondItem="iDm-Uy-q8n" secondAttribute="trailing" id="3OU-Q9-FxD"/>
+                                                <constraint firstItem="iDm-Uy-q8n" firstAttribute="leading" secondItem="AfW-Ve-nGX" secondAttribute="leading" id="EZN-DE-7lN"/>
+                                                <constraint firstItem="iDm-Uy-q8n" firstAttribute="top" secondItem="AfW-Ve-nGX" secondAttribute="top" id="ccQ-X3-9Fr"/>
+                                                <constraint firstAttribute="bottom" secondItem="iDm-Uy-q8n" secondAttribute="bottom" id="nyM-6y-DAQ"/>
+                                            </constraints>
+                                        </view>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="rbQ-5G-zr8"/>
+                                    <barButtonItem title="Mode" id="hID-no-gPl">
+                                        <connections>
+                                            <action selector="modeButtonTapped:" destination="aTU-AV-gOg" id="zxy-gt-oIP"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem width="42" style="plain" systemItem="fixedSpace" id="PVk-Ok-Zb2"/>
+                                </items>
+                            </toolbar>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="umr-Q2-zU9">
+                                <rect key="frame" x="0.0" y="774" width="414" height="44"/>
+                                <items>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="LMf-qC-QSu"/>
+                                    <barButtonItem title="AM" id="SKA-Sg-ke6">
```

Comment:
A titled bar button item should be used for something like this since the items still act like buttons. Tap on and hold on either AM or PM to see what I mean. Instead, could you use bar button items labels instead?